pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "nist-1500-examples"
+version = "0.0.1"
+description = "NIST 1500 examples"
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+"electos.nist-datamodels" = {path = ".local/tmp/electos.nist_datamodels-0.0.4-py3-none-any.whl"}
```

Comment:
This isn't portable; I can't get poetry to install the package

Reply:
@cwulfman Thanks for this note, saved me a lot of time. Here are the commands I ran:

To work on branch `precinct-specific-edfs`

1. Create a fork in GitHub
2. Clone the fork (I use `~/repos/forks/`)
3. In the project dir, `git checkout precinct-specific-edfs`
4. Run `mkdir -p .local/tmp`

Before installing with poetry,

1. Clone [ion-oset/nist-data-models: Dataclasses for NIST SP-1500 schemas](https://github.com/ion-oset/nist-data-models) into ~/repos/ion-oset/nist-data-models (my suggested path)
2. In the newly-cloned project, activate the python branch: `git checkout python/main`
3. Build the wheel file: `poetry build`
4. Copy the wheel to the fork: `cp dist/electos.nist_datamodels-0.0.4-py3-none-any.whl ~/repos/forks/nist-1500-examples/.local/tmp/`

Now, run `poetry install` on the nist-1500-examples fork.

Reply:
That's a bug, which I fixed but forgot to push. It's supposed to pull a specific tag from the NIST data models repository

```
```

See: 52a2382a which is now part of the PR.